src/main/java/core/basesyntax/SupplyFruitOperation.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import java.util.Map;
+
+public class SupplyFruitOperation extends FruitOperation {
+    public SupplyFruitOperation(OperationType type, Transaction fruit) {
+        super(type, fruit);
+    }
+
+    @Override
+    public Map<String, Transaction> execute(int totalQuantity,
+                                            Map<String, Transaction> storage) {
```

Comment:
please see Thursday's lesson. I am not passing the storage as a method parameter